book/website/reproducible-research/overview/overview-resources.md

```diff
@@ -21,6 +21,30 @@ If you want to start learning hands-on practices, we recommend reading the {ref}
 (rr-overview-resources-addmaterial)=
 ## Additional Material
 
+(rr-overview-resources-commandline)=
+### Using the command line
+
+While most of what this book is about does not require the use of the command line, it often actually makes things easier. 
+Even when your visual abilities allows you to use efficiently your mouse, writing commands in the terminal/powershell/command is not that difficult.
```

Comment:
```suggestion
Writing commands in the terminal, PowerShell, or command prompt provides an alternative means of interacting with the computer. It offers several advantages and can be considered a beneficial approach for various reasons:
- By directly entering commands, tasks can be performed efficiently without relying on graphical user interfaces (GUIs) or navigating through multiple menus.
- It provides higher level of flexibility and control as the user can execute complex tasks, automate repetitive actions, and customise commands according to specific needs.
- The terminal is available on various operating systems. Hence, knowing how to navigate and use the command-line interface ensures a consistent approach across different platforms.
- Familiarity with the terminal enhances the user's understanding of the underlying operating system and improves their technical skills. 
- When encountering issues or performing system diagnostics, the command-line utilities and diagnostic commands allow the user to investigate and resolve problems at a deeper level than GUI-based tools.
```

I would rephrase to not limit only to visual abilities as people also use speech output systems (like screen-readers), braille displays, and other assistive technologies to use computers. Also I won't comment whether it is a simple way or not rather, I would highlight what are the advantages of using it.

Reply:
Makes sense!